### PR TITLE
[stable/mysql] Add OWNERS file and olemarkus as maintainer

### PR DESCRIPTION
**Pull request by olemarkus**
**What this PR does / why we need it**:
This feedback adds an OWNERS file to the stable/mysql chart as well as adding olemarkus as an additional maintainer.

**Special notes for your reviewer**:
OK'ed by @viglesiasce via mail.

### CODE REVIEW

**Comment by mattfarina** [comment]
@viglesiasce Since you are the current owner can you comment on this PR?

File: stable/mysql/.helmignore [hunk 1]
```diff
@@ -1 +1,2 @@
 .git
+OWNERS
```
Comment (viglesiasce) [inline stable/mysql/.helmignore:2]: I don't believe this is necessary

**Review by viglesiasce — CHANGES_REQUESTED** [review]
Looks good. Can you remove the .helmignore change?

File: stable/mysql/.helmignore (hunk 1 above)
Comment (olemarkus) [reply stable/mysql/.helmignore:2]: Well. Documentation recommends it: https://github.com/helm/charts/blob/master/README.md#owning-and-maintaining-a-chart

Comment (viglesiasce) [reply stable/mysql/.helmignore:2]: Interesting! Learning a bit everyday.

**Review by viglesiasce — APPROVED** [review]

**Pull request merged**